jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardWithGridMarkupTests.java

```diff
@@ -0,0 +1,156 @@
+        baseValidation(cardWithGridMarkup.listCard.get(2).title);
+        baseValidation(cardWithGridMarkup.listCard.get(2).paragraph);
+        baseValidation(cardWithGridMarkup.listCard.get(2).button);
+    }
+
+    @Test
+    public void baseValidationCardWithGridMarkup22Test() {
+        baseValidation(cardWithGridMarkup.listCard.get(3));
+        baseValidation(cardWithGridMarkup.listCard.get(3).title);
+        baseValidation(cardWithGridMarkup.listCard.get(3).paragraph);
+        baseValidation(cardWithGridMarkup.listCard.get(3).button);
+    }
+
+    //test title
+    @Test
+    public void getTitleTextCardWithGridMarkup11Test() {
+        cardWithGridMarkup.listCard.get(0).title.is().text(textTitleCardWithGridMarkup11);
+    }
+
+    @Test
+    public void getTitleTextCardWithGridMarkup12Test() {
+        cardWithGridMarkup.listCard.get(1).title.is().text(textTitleCardWithGridMarkup12);
+    }
+
+    @Test
+    public void getTitleTextCardWithGridMarkup21Test() {
+        cardWithGridMarkup.listCard.get(2).title.is().text(textTitleCardWithGridMarkup21);
+    }
+
+    @Test
+    public void getTitleTextCardWithGridMarkup22Test() {
+        cardWithGridMarkup.listCard.get(3).title.is().text(textTitleCardWithGridMarkup22);
+    }
+    
+    //test paragraph
+    @Test
+    public void getParagraphTextCardWithGridMarkup11Test() {
+        cardWithGridMarkup.listCard.get(0).paragraph.is().text(textParagraphCardWithGridMarkup11);
+    }
+
+    @Test
+    public void getParagraphTextCardWithGridMarkup12Test() {
+        cardWithGridMarkup.listCard.get(1).paragraph.is().text(textParagraphCardWithGridMarkup12);
+    }
+
+    @Test
+    public void getParagraphTextCardWithGridMarkup21Test() {
+        cardWithGridMarkup.listCard.get(2).paragraph.is().text(textParagraphCardWithGridMarkup21);
+    }
+
+    @Test
+    public void getParagraphTextCardWithGridMarkup22Test() {
+        cardWithGridMarkup.listCard.get(3).paragraph.is().text(textParagraphCardWithGridMarkup22);
+    }
+
+    //test button
+    @Test
+    public void getButtonTextCardWithGridMarkup11Test() {
+        cardWithGridMarkup.listCard.get(0).button.is().text(textButtonCardWithGridMarkup11);
+        cardWithGridMarkup.listCard.get(0).button.click();
+        validateAlert(is(textAlert));
+        cardWithGridMarkup.listCard.get(0).button.is()
+                .enabled()
+                .displayed();
+    }
+
+    @Test
+    public void getButtonTextCardWithGridMarkup12Test() {
+        cardWithGridMarkup.listCard.get(1).button.is().text(textButtonCardWithGridMarkup12);
+        cardWithGridMarkup.listCard.get(1).button.click();
+        validateAlert(is(textAlert));
+        cardWithGridMarkup.listCard.get(1).button.is()
+                .enabled()
+                .displayed();
+    }
+
+    @Test
+    public void getButtonTextCardWithGridMarkup21Test() {
+        cardWithGridMarkup.listCard.get(2).button.is().text(textButtonCardWithGridMarkup21);
```

Comment:
allocate new method variable for: cardWithGridMarkup.listCard.get(2)

Reply:
still no changes here

Reply:
the same problem with display